### PR TITLE
Add a test for the expected number of processes in the container

### PR DESCRIPTION
**Pull request by JayH5**

### CODE REVIEW

File: example/test.sh [hunk 1]
```diff
@@ -76,6 +86,10 @@ curl -fsI http://localhost:$WEB_PORT/static/admin/img/search.svg | fgrep -v 'Cac
 
 # Celery tests
 # ############
+# Assert the expected number of processes are running in the web container
+[ "$(compose_cmd exec web ps ax --no-headers | grep -v 'ps ax' | wc -l | tr -d ' ')" \
```
Comment (bearnard) [inline example/test.sh:90]: does wc output spaces?

**Comment by bearnard** [comment]
👍

File: example/test.sh (hunk 1 above)
Comment (JayH5) [reply example/test.sh:90]: It does on macOS 😿

Comment (bearnard) [reply example/test.sh:90]: ah cool.

**Pull request merged**